### ISSUE TITLE
Can't retrieve stats for a public github repo

### ISSUE THREAD

**Issue by abailly-iohk** [bbougon/git-stats#25]
**Describe the bug**

Running the following command from `master`:

```
gitflows-stats github $(cat github-token) input-output-hk hydra 2023-01-01 2023-06-30
```

Yields the following somewhat cryptic output:

```
Paginate | ████████████████████████████████████████ | 9/9
Something wrong happened:
TypeError: Cannot read properties of undefined (reading 'build')
```

I generated a new PAT and checked it's working with a raw curl on the GH API.

**Comment by abailly** [comment]
It does not happen when I add `--format html`

**Comment by bbougon** [comment]
Hello @abailly, 

Thank you for your report.

I ran into that yesterday. This is related to this [part](https://github.com/bbougon/git-stats/blob/44b1e10fab95da8b5fc26dd6ff25c59ee754f8f1/src/infrastructure/writer/ConsoleWriter.ts#L93) where I missed the new issues events:

```
private static statisticsBuilder: Map<string, StatisticBuilder<any>> = new Map<string, StatisticBuilder<any>>([
    ["mergedEventsStatistics", new MergedEventsStatisticByPeriodBuilder()],
    ["mergeEvents", new MergeEventsStatisticBuilder()],
    ["cumulativeStatistics", new CumulativeStatisticBuilder()],
  ]);

  constructor(private readonly key: string, private readonly statistics: GitStatistics) {}

  build(): object {
    return ConsoleContentBuilder.statisticsBuilder.get(this.key).build(this.statistics);
  }
}
```

 I'll fix it during lunch break.

**Comment by bbougon** [comment]
Bug is fixed on release 0.2.7.
